packages/gatsby/src/redux/actions.js

```diff
@@ -1134,11 +1134,11 @@ actions.addThirdPartySchema = (
 }
 
 /**
- * All defined actions.
+ * All action creators wrapped with a dispatch.
  */
 exports.actions = actions
 
 /**
- * All action creators wrapped with a dispatch.
+ * All action creators wrapped with a dispatch. - *DEPRECATED*
```

Comment:
Are we doing JSDoc in this repo? If so, I think a `@deprecated` tag would help here.